LayoutTests/fast/scrolling/ios/scroll-anchoring-momentum-scroll.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html> <!-- webkit-test-runner [ useFlexibleViewport=true AsyncOverflowScrollingEnabled=true AsyncFrameScrollingEnabled=true ] -->
```

Comment:
Is `useFlexibleViewport=true` necessary?

Reply:
Removed.